ecommerce/frontend/src/components/layout/Content.js

```diff
@@ -28,7 +29,7 @@ export const Content = () => {
     return (
         <Layout.Content className="content">
             <CategoryBar />
-            {location.pathname.startsWith('/profile') || <IsVerifiedNotification />}
+            {isUser && user.is_verified && <IsVerifiedNotification />}
 
```

Comment:
the reason we have the `location.pathname` is because when you are in the profile age the notification will appear twice, both in Content and in Profile page. maybe you can add those `isUser...` conditions?

Reply:
I hadn't seen this component.
![image](https://user-images.githubusercontent.com/44057640/105630080-e01b8e00-5e57-11eb-9923-6870ce682a4d.png)